src/main/java/org/randywebb/wlts/ldstools/rest/LdsToolsClient.java

```diff
@@ -78,6 +78,7 @@ public static CloseableHttpClient getHttpClient() {
         @throws AuthenticationException In the event username or password is incorrect
     */
     private void signIn(String user, String password) throws AuthenticationException {
+        final int httpOk = 200;
```

Comment:
Meh, I guess I speak fluent HTTP Status Codes :) It's OK, though I'd name it as a Constant "HTTP_OK"

Reply:
The linter catches "magic numbers" I understood 200 perfectly as well.

Reply:
Fair enough. I agree that it is more readable.
You define this in 2 different methods in the code. I don't think we warrant a Constants class or anything though. I remember that the HTTP libraries have these constants already defined, but I don't recall where offhand.